src/schema/v2/artwork/artworkVisibility.ts

```diff
@@ -0,0 +1,14 @@
+import { GraphQLEnumType } from "graphql"
+
+export const ArtworkVisibility = new GraphQLEnumType({
+  name: "Visibility",
+  values: {
+    UNLISTED: { value: "unlisted" },
```

Comment:
It's minor, but can remove a bit of duplication here by rearranging the variables in the file: 

`value: ArtworkVisibilityEnumValues.UNLISTED`  

(and so on)

Reply:
Nice catch! 🙏🏻